.github/workflows/release.yaml

```diff
@@ -13,9 +13,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Build Windows logger agent
-        run: make build-logger-windows
+        run: $env:VERSION=$env:GITHUB_REF.substring(10, $env:GITHUB_REF.length-10); make build-logger-windows
```

Comment:
Even though this `$env:VERSION=$env:GITHUB_REF.substring(10, $env:GITHUB_REF.length-10)` clause repeats a total of four times in this file, this seems to be the most straightforward way of doing this.

I attempted [this approach](https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-environment-variable) and didn't have much luck.